lxd/main_init_interactive.go

```diff
@@ -226,7 +226,19 @@ func (c *cmdInit) askClustering(config *cmdInitData, d lxd.InstanceServer) error
 					certDigest := shared.CertFingerprint(cert)
 					fmt.Printf("Cluster fingerprint: %s\n", certDigest)
 					fmt.Printf("You can validate this fingerprint by running \"lxc info\" locally on an existing node.\n")
-					if !cli.AskBool("Is this the correct fingerprint? (yes/no) [default=no]: ", "no") {
+					validator := func(input string) error {
+						if input == certDigest {
+							return nil
+						} else if shared.StringInSlice(strings.ToLower(input), []string{"yes", "y"}) {
+							return nil
+						} else if shared.StringInSlice(strings.ToLower(input), []string{"no", "n"}) {
+							return nil
+						}
+						return fmt.Errorf("Please type 'y', 'n' or the fingerprint:")
```

Comment:
I'd put a line break before this.

I'd also turn the error into `Not yes/no or fingerprint`. So that it renders as:
```
Invalid input: Not yes/no or fingerprint
```

Reply:
Do you mean a line break before the error message? That would cause it to print
```
Invalid input:
Not yes/no or fingerprint
```

I could add the line break into AskString, but that would affect more than just this line.

Or did you mean a line break in the actual code?

(Also sorry about the title, I'm a bit out of practice with PRs, my brain ignored the title entry entirely!)

Reply:
No, line break in the code to separate the AskString call from the block above it.

Reply:
We usually put line breaks before/after if/for/switch type blocks (there are some exception like a variable being defined before a loop for use only within that loop, then we'd keep that definition stuck to the for statement).